api/src/main/java/de/adorsys/keymanagement/api/keystore/KeyStoreView.java

```diff
@@ -35,7 +35,23 @@
      */
     <Q> AliasView<Q> allAliases();
 
+    /**
+     * Get source of keys
+     * @return object which have access to keys and can operate with them
+     */
     KeySource source();
+
+    /**
+     * Generates keySet from all existing in key source keys
```

Comment:
`Clones current KeySource into KeySet in a way, that all keys will be materialized in KeySet. You can view it as generates keySet from all existing in key source keys`

Reply:
done